src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/InformationList.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+
+namespace Tizen.MachineLearning.Inference
+{
+    public class MlInformationList {
+        private IntPtr _handle = IntPtr.Zero;
+
+        internal MlInformationList(IntPtr handle) {
+            NNStreamer.CheckNNStreamerSupport();
+            _handle = handle;
+        }
+
+        ~MlInformationList() {
+            NNStreamerError ret = Interop.Util.DestroyInformationList(_handle);
+            NNStreamer.CheckException(ret, "Failed to destroy the information list");
+        }
+
+        public MlInformation GetInformation(int index)
+        {
+            IntPtr value;
```

Comment:
ditto init value
IntPtr value = IntPtr.Zero;

Reply:
Updated
Thank you :)